### PR TITLE
🌐 Add Traditional Chinese translation for `docs/zh-hant/docs/virtual-environments.md`

### PR DESCRIPTION
**Pull request by Vincy1230**
Add Traditional Chinese translation for virtual-environments.md
為 virtual-environments（虛擬環境）一章加入翻譯

### CODE REVIEW

File: docs/zh-hant/docs/virtual-environments.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+
+///
+
+## 檢查虛擬環境是否啟動
+
+檢查虛擬環境是否啟動（前面的指令是否生效）。
+
+/// tip
+
+這是 **可選的**，但這是一個很好的方法，可以 **檢查** 一切是否按預期工作，以及你是否使用了你打算使用的虛擬環境。
```
Comment (celestywang) [inline docs/zh-hant/docs/virtual-environments.md:179]: how about 非必需的?

File: docs/zh-hant/docs/virtual-environments.md [hunk 2]
```diff
@@ -0,0 +1,844 @@
...
+flowchart LR
+    subgraph global[全域環境]
+        harry-1[harry v1]
+    end
+    subgraph stone-project[專案 philosophers-stone]
+        stone(philosophers-stone) -->|需要| harry-1
+    end
+```
+
+但如果你想運行 `prisoner-of-azkaban`，你需要卸載 `harry` 版本 `1` 並安裝 `harry` 版本 `3`（或者只要你安裝版本 `3`，版本 `1` 就會自動卸載）。
```
Comment (KNChiu) [inline docs/zh-hant/docs/virtual-environments.md:490]: 一般來說  `uninstall` 會翻譯成 `解除安裝` 或是 `移除`

Comment (Vincy1230) [reply docs/zh-hant/docs/virtual-environments.md]: 好的，已修正

Comment (KNChiu) [reply docs/zh-hant/docs/virtual-environments.md]: 辛苦了

**Review by KNChiu — APPROVED** [review]
LGTM

File: docs/zh-hant/docs/virtual-environments.md [hunk 3]
```diff
@@ -0,0 +1,844 @@
...
+* 安裝**套件**
+* 為你的專案管理套件的**依賴和版本**
+* 確保你有一個**精確**的套件和版本集合來安裝，包括它們的依賴項，這樣你可以確保專案在生產環境中運行的狀態與開發時在你的電腦上運行的狀態完全相同，這被稱為**鎖定**
+* 還有很多其他功能
+
+## 結論
+
+如果你讀過並理解了所有這些，現在**你對虛擬環境的了解已超過許多開發者**。🤓
+
+未來當你調試看起來複雜的問題時，了解這些細節很可能會有所幫助，你會知道**它是如何在底層運作的**。😎
```
Comment (mattwang44) [inline docs/zh-hant/docs/virtual-environments.md:844]:
```suggestion
未來當你為看起來複雜的問題除錯時，了解這些細節很可能會有所幫助，你會知道**它是如何在底層運作的**。😎
```

File: docs/zh-hant/docs/virtual-environments.md [hunk 4]
```diff
@@ -0,0 +1,844 @@
...
+$ python main.py
+
+Hello World
+```
+
+</div>
+
+## 設定編輯器
+
+你可能會用到編輯器（即 IDE —— 譯者註），請確保設定它使用你建立的相同虛擬環境（它可能會自動偵測到），以便你可以獲得自動完成和內聯錯誤提示。
```
[the remark this answers is not in the document]
Comment (Vincy1230) [reply docs/zh-hant/docs/virtual-environments.md:395]: 已刪去。中國大陸的開發者常會混用這兩個概念，在同時翻譯兩個語言的時候沒有注意到差異。

File: docs/zh-hant/docs/virtual-environments.md [hunk 5]
```diff
@@ -0,0 +1,844 @@
...
+
+<div class="termy">
+
+```console
+$ deactivate
+```
+
+</div>
+
+這樣，當你執行 `python` 時，它不會嘗試從已安裝套件的虛擬環境中執行。（即，它將不再會嘗試從虛擬環境中執行，也不會使用其中安裝的套件。 —— 譯者註）
```
[the remark this answers is not in the document]
Comment (Vincy1230) [reply docs/zh-hant/docs/virtual-environments.md:420]: OK

**Comment by Vincy1230** [comment]
搞定了 @mattwang44

File: docs/zh-hant/docs/virtual-environments.md [hunk 6]
```diff
@@ -0,0 +1,844 @@
...
+
+/// info
+
+這個頁面將教你如何使用**虛擬環境**以及了解它們的工作原理。
+
+如果你計畫使用一個**可以為你管理一切的工具**（包括安裝 Python），試試 <a href="https://github.com/astral-sh/uv" class="external-link" target="_blank">uv</a>。
+
+///
+
+## 創建一個專案
```
Comment (mattwang44) [inline docs/zh-hant/docs/virtual-environments.md:29]: 「創建」一詞多次出現，要請你再檢查全文一遍

File: docs/zh-hant/docs/virtual-environments.md [hunk 7]
```diff
@@ -0,0 +1,844 @@
...
+
+///
+
+## 檢查虛擬環境是否啟動
+
+檢查虛擬環境是否啟動（前面的指令是否生效）。
+
+/// tip
+
+這是 **非必需的**，但這是一個很好的方法，可以 **檢查** 一切是否按預期工作，以及你是否使用了你打算使用的虛擬環境。
```
Comment (mattwang44) [inline docs/zh-hant/docs/virtual-environments.md:179]: 中文間不需空格、中文與英文間需要空格
```suggestion
這是**非必需的**，但這是一個很好的方法，可以**檢查**一切是否按預期工作，以及你是否使用了你打算使用的虛擬環境。
```

**Review by mattwang44 — COMMENTED** [review]
我 review 時比較偷懶，同樣的問題只會留一次 comment，想請你再依先前的建議檢查全文是否還有同樣的問題

File: docs/zh-hant/docs/virtual-environments.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mattwang44) [reply docs/zh-hant/docs/virtual-environments.md]: 原來你同時發了兩邊的 PR 👍🏽

File: docs/zh-hant/docs/virtual-environments.md (hunk 7 above)
Comment (Vincy1230) [reply docs/zh-hant/docs/virtual-environments.md]: 搞定啦

File: docs/zh-hant/docs/virtual-environments.md [hunk 8]
```diff
@@ -0,0 +1,844 @@
...
+
+/// tip
+
+當你需要安裝或升級套件時，執行本操作**一次**；
+
+如果你需要再升級版本或新增套件，你可以**再次執行此操作**。
+
+///
+
+### 直接安裝包
```
Comment (mattwang44) [inline docs/zh-hant/docs/virtual-environments.md:297]:
```suggestion
### 直接安裝套件
```

**Review by mattwang44 — COMMENTED** [review]
should be the last one

File: docs/zh-hant/docs/virtual-environments.md (hunk 8 above)
Comment (Vincy1230) [reply docs/zh-hant/docs/virtual-environments.md]: OK done :)

**Review by mattwang44 — APPROVED** [review]
nice work!

File: docs/zh-hant/docs/virtual-environments.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (mattwang44) [reply docs/zh-hant/docs/virtual-environments.md]: 好奇想請教你是參考哪一份對照表呢？
我自己是會參考這兩份（來自 [Python 文件繁中翻譯 repo wiki](https://github.com/python/python-docs-zh-tw/wiki/___%E8%A9%B2%E7%BF%BB%E6%88%90%E4%BB%80%E9%BA%BC%EF%BC%9F)）：
- [L10N Taiwan - 詞彙對照表](https://hackmd.io/@l10n-tw/glossaries)
- [維基教科書 - 大陆台湾计算机术语对照表](https://zh.wikibooks.org/zh/%E5%A4%A7%E9%99%86%E5%8F%B0%E6%B9%BE%E8%AE%A1%E7%AE%97%E6%9C%BA%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7%E8%A1%A8)

[the remark this answers is not in the document]
Comment (Vincy1230) [reply docs/zh-hant/docs/virtual-environments.md]: the last one

File: docs/zh-hant/docs/virtual-environments.md [hunk 9]
```diff
@@ -0,0 +1,844 @@
...
+    end
+    subgraph stone-project[專案 philosophers-stone]
+        stone(philosophers-stone) -.-x|⛔️| harry-1
+    end
+    subgraph azkaban-project[專案 prisoner-of-azkaban]
+        azkaban(prisoner-of-azkaban) --> |需要| harry-3
+    end
+```
+
+/// 提示
```
Comment (alejsdev) [inline docs/zh-hant/docs/virtual-environments.md:519]: This is not rendering properly. Can you check and fix it? (you can validate with the preview)
Thank you!

Comment (Vincy1230) [reply docs/zh-hant/docs/virtual-environments.md]: Okey... It should be able to work now 🤔

**Review by yychanlee — APPROVED** [review]

**Comment by Vincy1230** [comment]
Say a digression:
Why do Github Actions often get stuck on `smokeshow / coverage Expected — Waiting for status to be reported` when I commit pull requests for FastAPI? What's this 🤔

**Comment by Vincy1230** [comment]
let me try to sync the repository again to trigger the Actions again 🤔, but why 🤔

**Comment by Vincy1230** [comment]
Now it works, but... why 😭

**Comment by tiangolo** [comment]
> Now it works, but... why 😭

It seems to be a hiccup on GitHub's side, related to how it stores the coverage files. It's not on your side. :sweat_smile: Don't worry about these cases. We just need to re-run the tests so that Smokeshow can collect the coverage files that were somehow lost by GitHub Actions.

**Comment by alejsdev** [comment]
Thanks for your help! @Vincy1230 :nerd_face: 
And thanks to everyone for the reviews :mag:

**Pull request merged**